indico/modules/events/sessions/blueprint.py

```diff
@@ -52,6 +54,15 @@
 _bp.add_url_rule('/manage/sessions/<int:session_id>/acl', 'acl', RHSessionACL)
 _bp.add_url_rule('/manage/sessions/<int:session_id>/acl-message', 'acl_message', RHSessionACLMessage)
 
+# Session types
+_bp.add_url_rule('/manage/sessions/types/', 'manage_types', RHManageSessionTypes)
+_bp.add_url_rule('/manage/sessions/types/create', 'create_type', RHCreateSessionType,
```

Comment:
what maxlinelength are you using in your IDE?

Reply:
120, but it's true that I didn't need to wrap this one :)